INSTALL

```diff
@@ -51,24 +51,35 @@ pip install . \
   --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-master/constraints-3.6.txt"
 
 
-By default `pip install` in Airflow 2.0 installs only the provider packages that are needed by the extras,
-however if you want to install all providers (which was default behaviour in 1.10.*)
-you can do it by setting environment variable INSTALL_PROVIDERS_FROM_SOURCES to `true`.
+By default `pip install` in Airflow 2.0 installs only the provider packages that are needed by the extras and
+install them as packages from PyPI rather than from local sources:
 
 
-INSTALL_PROVIDERS_FROM_SOURCES="true" pip install . \
+pip install . \
   --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-master/constraints-3.6.txt"
 
 
-You can also install airflow in "editable mode" (with -e) flag and then provider packages will be
-available, because they are used directly from the airflow sources:
-
+You can also install airflow in "editable mode" (with -e) flag and then provider packages are
+available directly from the sources (and the provider packages installed from PyPI are uninstalled in
+order to avoid having providers in two places. This is useful if you want to develop providers:
 
 pip install -e . \
   --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-master/constraints-3.6.txt"
 
+You can als skip installing provider packages from PyPI by setting INSTALL_PROVIDERS_FROM_SOURCE to "true".
+In this case Airflow will be installed in non-editable mode with all providers installed from the sources.
+Additionally provider.yaml files will also be copied to providers folders which will make the providers
```

Comment:
The `provider.yaml` file is not described here so users will not know what it means, we should either said what it is, or remove the reference.

Btw. isn't it `providers.yaml` (plural)?

Reply:
Nope. It's provider.yaml

Reply:
Good point with the description of provider.yaml file. I added a chapter about it in the 'Provider Packages' documentation  and referred to it.